metadata/src/lib.rs

```diff
@@ -0,0 +1,18 @@
+//! Metadata information and types for ceramic
+
+include!(concat!(env!("OUT_DIR"), "/built.rs"));
```

Comment:
Why can't we just use a normal `import` here.
```rust
mod built;
pub use built::PKG_VERSION
```

Why don't we check in the build.rs output?
On the one hand it is generated data but the other code reads more easaly if I know what is in the file it is getting values from.